app/repositories/CampaignContentRepository.scala

```diff
@@ -17,11 +23,25 @@ object CampaignContentRepository {
   private def getItemsForCampaignId(campaignId: String): List[Item] =
     Dynamo.campaignContentTable.query("campaignId", campaignId).toList
 
-  def getContentForCampaign(campaignId: String): List[ContentItem] =
-    getItemsForCampaignId(campaignId).map { ContentItem.fromItem }
+  def getContentForCampaign(campaignId: String): Either[CampaignCentralApiError, List[ContentItem]] = {
+    val result: List[Either[CampaignCentralApiError, ContentItem]] = getItemsForCampaignId(campaignId).map {
+      ContentItem.fromItem
+    }
+    result.sequence
```

Comment:
This `sequence` only works on the `Right`; not saying to change it but highlighting that you don't collect `Left` at all when using `sequence` from `cats`.

Reply:
Yes, am aware. Didn't want to introduce `Validated`. This gives enough feedback to know that something is broken and a reason why.